renderdoc/driver/d3d12/d3d12_command_list_wrap.cpp

```diff
@@ -29,6 +29,23 @@
 #include "d3d12_command_queue.h"
 #include "d3d12_debug.h"
 
+#define EXECUTE_COMMAND_WITH_ACTION_CALLBACK(dispatch, ...)       \
+  uint32_t eventId = m_Cmd->HandlePreCallback(list, dispatch);    \
+  __VA_ARGS__                                                     \
+  if(eventId && m_Cmd->m_ActionCallback->PostDraw(eventId, list)) \
+  {                                                               \
+    __VA_ARGS__                                                   \
+    m_Cmd->m_ActionCallback->PostRedraw(eventId, list);           \
+  }
```

Comment:
I'd rather not have a complex parameterised macro like this. There's a bug here that dispatches should be using `PostDispatch` and `PostRedispatch` instead of `PostDraw` (which should only be for drawcalls, to allow callbacks to filter between them as needed)

Reply:
As mentioned above please remove these macros and leave the code as-is in the places where it's called, the same way that you can see is done in the vulkan code.

Reply:
> As mentioned above please remove these macros and leave the code as-is in the places where it's called, the same way that you can see is done in the vulkan code.

Done.